package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-find-rules",
-  "version": "3.1.0",
+  "version": "3.1.1",
```

Comment:
Right but you can fast forward merge an approved PR that does nothing but the version bump, which is effectively a direct commit to master.